pages/components/modal/modal.css

```diff
@@ -0,0 +1,86 @@
+:root {
+    --modal-bg-color: rgb(0, 0, 0);
+    --modal-bg-opacity: rgba(0, 0, 0, 0.4);
+    --close-black: black;
+}
+
+.modal-container {
+    width: 60vw;
+    margin: 1rem;
+    padding: 1rem;
+}
+
+
+.modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    padding-top: 100px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: var(--modal-bg-color);
+    background-color: var(--modal-bg-opacity);
+}
+
+.modal-content {
+    background-color: var(--primary-light);
+    text-align: center;
+    margin: auto;
+    padding: 20px;
+    width: 35%;
+    box-shadow: 3px 3px 15px var(--modal-bg-color);
+}
+
+#open-modal {
```

Comment:
Why you are grabbing the element by its id, use classes in place in place of this, as they can be used in multiple elements.

Reply:
Thanks for pointing out. Changed it to class now. please chack.